pkg/skaffold/build/buildpacks/types.go

```diff
@@ -22,12 +22,14 @@ import "github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker"
 type Builder struct {
 	localDocker docker.LocalDaemon
 	pushImages  bool
+	devMode     bool
```

Comment:
why did you decide to put devMode live on the Builder and not on the artifact configuration?

Reply:
Hmmm... I got confused and thought that "devMode" is specific to buildpacks, however it just signifies that we started `skaffold dev`. You can ignore my comment. Something around how we wire through our code, when we introduce new flags / options from runcontext still rubs me the wrong way.